api-reference/10 UI Components/dxDataGrid/1 Configuration/export/formats.md

```diff
@@ -6,7 +6,368 @@ default: 'DataGrid'
+
+    import { DxDataGridModule } from 'devextreme-angular';
+
+    @NgModule({
+        declarations: [
+            AppComponent
+        ],
+        imports: [
+            BrowserModule,
+            DxDataGridModule
+        ],
+        providers: [ ],
+        bootstrap: [AppComponent]
+    })
+    export class AppModule { }
+
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <div>
+            <DxDataGrid ...
+                @exporting="onExporting"
+            >
+                <!-- ... -->
+                <DxExport
+                    :enabled="true"
+                    :formats="['pdf']"
+                />
+            </DxDataGrid>
+        </div>
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.light.css';
+    import { DxDataGrid, DxExport } from 'devextreme-vue/data-grid';
+    import { jsPDF } from 'jspdf';
+    import { exportDataGrid } from 'devextreme/pdf_exporter';
+
+    export default {
+        components: {
+            DxDataGrid
+        }
+        data() {
+            return {
+                // ...
+            };
+        },
+        methods: {
+            onExporting(e) {
+                const doc = new jsPDF();
+                exportDataGrid({
+                    jsPDFDocument: doc,
+                    component: e.component,
+                }).then(() => {
+                    doc.save('Companies.pdf');
+                });
+            },
+        },
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+    import 'devextreme/dist/css/dx.light.css';
+    import DataGrid, { Export } from 'devextreme-react/data-grid';
+    import { jsPDF } from 'jspdf';
+    import { exportDataGrid } from 'devextreme/pdf_exporter';
+
+    const exportFormats = ['pdf'];
+
+    export default function App() {
+        const onExporting = React.useCallback((e) => {
+            const doc = new jsPDF();
+            exportDataGrid({
+                jsPDFDocument: doc,
+                component: e.component,
+            }).then(() => {
+                doc.save('Companies.pdf');
+            });
+        });
+        return (
+            <React.Fragment>
+                <div>
+                    <DataGrid ...
+                        onExporting={onExporting}
+                    >
+                        {/* ... */}
+                        <Export enabled={true} formats={exportFormats}>
+                    </DataGrid>
+                </div>
+            </React.Fragment>
+        );
+    }
+
+---
+
+You can specify multiple formats: `['xlsx', 'pdf', string]`. The `string` format specifies the custom text shown in the **Export** button.
+
+The example below shows how to export DataGrid to CSV format.
+
+---
+##### jQuery
+
+    <!--JavaScript-->
+    $(function(){
+        const dataGrid = $('#gridContainer').dxDataGrid({
+            // ...
+            export: {
+                enabled: true,
+                formats: ['csv'],
```

Comment:
Maybe better to add multiple formats to this example? It may be confusing for users how to find out which button ('xlsx' or 'pdf') was clicked in `onExporting` (they need to write `if (e.format === 'xlsx') {...} else if (e.format === 'pdf') {...}`

Reply:
Noted